src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,70 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String DELIMITER = ",";
+    private static final int FIELDS_SIZE = 2;
+    private static final int OPERATION_TYPE = 0;
+    private static final int AMOUNT = 1;
+    private String[] keys = new String[FIELDS_SIZE];
+    private Integer[] values = new Integer[FIELDS_SIZE];
+
     public void getStatistic(String fromFileName, String toFileName) {
+        StringBuilder inDatas = new StringBuilder();
+        inputFromFile(fromFileName, inDatas);
```

Comment:
return `inDatas` from `inputFromFile`, no need to pass it and change

Reply:
added supplementary object

Reply:
not fixed